source/components/table/table-component.js

```diff
@@ -2,6 +2,7 @@ import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import CSSModules from 'react-css-modules';
 import cx from 'classnames';
+import ResizeObserver from '../../polyfills/resize-observer-polyfill';
```

Comment:
Aqui a gente importa o polyfill. Ele segue a mesma sintaxe da api padrão do Resize Observer, então, se for implementado nos outros browsers, era só remover esse import aqui